Document max_token_header_size TCP listener configuration option (VAULT-43023) (#2882)
hashicorp/web-unified-docs: content/vault/v2.x/content/docs/internals/limits.mdx

the HTTP request itself, places an upper bound on any Transit operation,
and on the maximum size of any key-value secrets.

### Authentication token header size

The maximum size of the `X-Vault-Token` or `Authorization: Bearer` header is
controlled by the
[`max_token_header_size`](/vault/docs/configuration/listener/tcp#max_token_header_size)

> option in the listener stanza. The default is 8 KB (8192 bytes). This limit is a

Reviewer: ```suggestion
option in the listener stanza. The default is 8 KB (8192 bytes) as a
```
